dmcontent/content_loader.py

```diff
@@ -454,6 +455,9 @@ class ContentLoader(object):
     >>> # preload messages
     >>> loader.load_messages('framework-1', ['homepage_sidebar', 'dashboard'])
     >>>
+    >>> # preload metadata
+    >>> loader.load_metadata('framework-1', ['copying-services'])
+    >>>
```

Comment:
Might be worth adding a `loader.get_metatdata()` example as per manifest and message below.

Reply:
👍